CHANGELOG.asciidoc

```diff
@@ -18,17 +18,12 @@ endif::[]
 ===== Bug fixes
 ////
 
-=== Unreleased
+[[release-notes-0.x]]
+=== Android agent version 0.x
```

Comment:
Does this line have to be below every `[[release-notes-x.x]]`? Or is it just for the latest one? Because I noticed that other versions have something like `==== x.x.x = YYYY/MM/DD` instead.

Reply:
This is the heading that all 0.x releases will fall under. It allows us to do this: https://www.elastic.co/guide/en/apm/agent/nodejs/current/release-notes.html

Reply:
I see! thank you :)